doc/releases/release-notes-3.3.rst

```diff
@@ -1444,6 +1457,9 @@ Libraries / Subsystems
   * Added the linear range API to map values in a linear range to a range index
     :zephyr_file:`include/zephyr/sys/linear_range.h`.
 
+* DSP subsystem is now supported along with CMSIS as the default backend.
```

Comment:
"DSP" needs to be a dedicated section. This also needs to be alphabetically sorted

```suggestion
* DSP

  * Introduced DSP (digital signal processing) subsystem with CMSIS-DSP as the default backend.
  * CMSIS-DSP now supports all architectures supported by Zephyr.
```

Reply:
Done